main.cpp

```diff
@@ -55,9 +56,11 @@ struct endpoint_address {
 static struct opt {
     long unsigned baudrate;
     struct endpoint_address *ep_addrs;
+    bool report_msg_statistics;
```

Comment:
I think it's fine for now. But in the long run this would better be either
a) a signal you send via SIGUSR1 and it dumps statistics from an idler in the mainloop
or
b) we have a way to communicate with mavlink-router.... either a custom protocol over unix socket or dbus